### PR TITLE
[WIP]

### PR DESCRIPTION
**Pull request by bachand**

### CODE REVIEW

File: Rules.md [hunk 1]
```diff
@@ -1394,6 +1395,34 @@ Option | Description
 --- | ---
 `--trimwhitespace` | trim trailing space. "always" (default) or "nonblank-lines"
 
+## typeMarks
```
Comment (bachand) [inline Rules.md:1398]: @calda I was fiddling with this idea at one point but clearly did not get very far. The idea was to add autocorrect for https://github.com/airbnb/swift/issues/54. This tag has no expectations associated with it and is purely informational 👍

Comment (calda) [reply Rules.md:1398]: Omg nice! This would also be a great rule to bring back in the future.

Comment (bachand) [reply Rules.md:1398]: Agreed

Comment (calda) [reply Rules.md:1398]: [nicklockwood/SwiftFormat#747](https://github.com/nicklockwood/SwiftFormat/pull/747) :)

**Pull request closed without merging**